aleph/settings.py

```diff
@@ -201,6 +201,12 @@ def __init__(self) -> None:
         # XREF Model Selection
         self.XREF_MODEL = env.get("FTM_COMPARE_MODEL", None)
 
+        ###############################################################################
+        # Feature flags
+        self.ENABLE_EXPERIMENTAL_BOOKMARKS_FEATURE = env.get(
```

Comment:
Nitpick. The '_FEATURE' seems superfluous to me. "ENABLE_EXPERIMENTAL_BOOKMARKS" should be enought right?

Reply:
Makes sense, I’ll push an update! :)